services/brig/src/Brig/API/Internal.hs

```diff
@@ -86,8 +86,9 @@ sitemap = do
   post "/i/users" (continue createUserNoVerifyH) $
     accept "application" "json"
       .&. jsonRequest @NewUser
-  put "/i/self/email" (continue changeSelfEmailNoSendH) $
+  put "/i/self/email" (continue changeSelfEmailMaybeSendH) $
```

Comment:
eventhough we use an `internal` endpoint now; having it use the `zauth` header is still a bit akward to me.. but lets leave this; not for this PR to solve